### DISCUSSION TITLE
Ability to include tips?

### DISCUSSION THREAD

**Discussion by ruida-shen** [swellstores/community#327] [Help] [answered]
Hi there, is there a way to include tips during checkout?

**Comment by sags95** [comment, marked as answer]
Hey @ruida-shen, would you be able to expand on what you mean by tips? Would this be a field where the customer can add an additional tip at checkout?

This would not be possible with the Hosted Checkout unfortunately, unless you want to handle the logic in the cart page prior to checkout. If you're building a custom checkout then there might be a few ways for this to work. One way could be to create a "tip" product (you can set it as digital type to avoid fulfillment issues). If the customer wants to add a tip you can then add this product to the cart. For the tip amount, this will have to be set by updating the cart with the Backend API, as updating product pricing with the Frontend API is not possible.
